app/components/jobseekers/search_results/job_alerts_link_component.rb

```diff
@@ -1,4 +1,6 @@
 class Jobseekers::SearchResults::JobAlertsLinkComponent < ViewComponent::Base
+  attr_accessor :coordinates_present
```

Comment:
I don't think this attribute is ever set or read.

Reply:
good spot, i added this when trying something out